cram_urdf_projection/.gitignore

```diff
@@ -0,0 +1,5 @@
+*~
```

Comment:
Currently, this package is in the root of CRAM. We only add new directories to the root if it's really necessary. This package should rather be in cram_3d_world.